crates/bevy_text/src/error.rs

```diff
@@ -7,6 +7,7 @@ pub enum TextError {
     NoSuchFont,
     #[error("failed to add glyph to newly-created atlas {0:?}")]
     FailedToAddGlyph(GlyphId),
+    // TODO this is not used. Remove it.
```

Comment:
If it isn't used, then why not just remove it now? Is it too out of scope for the PR?

Reply:
It would introduce a breaking change, which means this can't be shipped in Bevy 0.9.